File: src/targets/python/requests/fixtures/jsonObj-null-value.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything"

payload = {"foo": None}
=======
url = "http://mockbin.com/har"

payload = { "foo": None }
>>>>>>> upstream/master
headers = {"content-type": "application/json"}

response = requests.post(url, json=payload, headers=headers)

<<<<<<< HEAD
print(response.text)
=======
print(response.json())
>>>>>>> upstream/master

File: src/targets/javascript/fetch/fixtures/jsonObj-null-value.js
<<<<<<< HEAD
const options = {
  method: 'POST',
  headers: {'content-type': 'application/json'},
  body: JSON.stringify({foo: null})
};

fetch('https://httpbin.org/anything', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));
=======
const url = 'http://mockbin.com/har';
const options = {
  method: 'POST',
  headers: {'content-type': 'application/json'},
  body: '{"foo":null}'
};

try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master
